build-system/amp4test.js

```diff
@@ -101,26 +106,31 @@ const bank = {};
  * Deposit a request. An ID has to be specified. Will override previous request
  * if the same ID already exists.
  */
-app.use('/request-bank/:bid/deposit/:id/', (req, res) => {
-  if (!bank[req.params.bid]) {
-    bank[req.params.bid] = {};
-  }
-  const key = req.params.id;
-  log('SERVER-LOG [DEPOSIT]: ', key);
-  if (typeof bank[req.params.bid][key] === 'function') {
-    bank[req.params.bid][key](req);
-  } else {
-    bank[req.params.bid][key] = req;
-  }
-  res.end();
-});
+app.use(
+    '/request-bank/:bid/deposit/:id/',
+    upload.array(),
```

Comment:
i suppose this is a no-op for non-form POST?

Reply:
Yup 😄 

<img width="1138" alt="screen shot 2019-02-20 at 1 00 51 pm" src="https://user-images.githubusercontent.com/1448289/53124194-9a3d4900-350f-11e9-8f2b-43dc2538d58b.png">